README.md

```diff
@@ -1,9 +1,13 @@
+<!-- markdownlint-disable MD041 -->
+|![Warning](https://upload.wikimedia.org/wikipedia/commons/thumb/1/17/Warning.svg/156px-Warning.svg.png) | This project is no longer supported. [hub.helm.sh](https://hub.helm.sh) now redirects to [artifacthub.io](https://artifacthub.io/). The source for that can be found at <https://github.com/artifacthub/hub>
```

Comment:
should we use the ⚠️ here instead of the .svg from wikimedia ?

Reply:
I just looked and current projects that have been deprecated use the wikimedia thing. I'm sure one person did it once and the rest of us copied it.